apps/comps/components/trades-table/index.tsx

```diff
@@ -69,12 +69,9 @@ export const TradesTable: React.FC<TradesTableProps> = ({
         cell: ({ row }) => (
           <div>
             <div className="text-primary-foreground text-sm">
-              {formatAmount(row.original.fromAmount)}{" "}
-              {"$" + row.original.fromTokenSymbol} →{" "}
-              {formatAmount(row.original.toAmount)}{" "}
-              {"$" + row.original.toTokenSymbol}
+              {"$" + formatAmount(row.original.fromAmount)}{" "}
               {row.original.fromTokenSymbol} →{" "}
-              {formatAmount(row.original.toAmount, 6)}{" "}
+              {"$" + formatAmount(row.original.toAmount)}{" "}
```

Comment:
the ticker should prefix the symbol, not the amount

<img width="207" height="124" alt="Image" src="https://github.com/user-attachments/assets/7e2aa226-12cd-4912-a39a-e2552c4e52a3" />